manpages/lumina-config.8

```diff
@@ -1 +1,25 @@
-Dummy file
+.Dd November 3, 2017
+.Dt LUMINA-CONFIG 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-config
+.Nd used to modify different desktop settings.
+
+.Sh SYNOPSIS
+.Nm
+
+.Sh DESCRIPTION
+.Nm
+opens the graphical utility to modify many different desktop settings.
+Settings include appearance, desktop defaults, interface
+configuration, system settings, and user settings.
```

Comment:
Suggested rework to put the list first, as it is the most important element of the sentence: "Appearance, desktop defaults, interface configuration, system settings, and user settings are adjustable." Lumina config has a large number of smaller adjustable elements - we could expand this with a list of every gui choice and a short description of the settings.